Framework/PythonInterface/plugins/algorithms/LoadNMoldyn3Ascii.py

```diff
@@ -21,10 +21,10 @@
 
 
 def _find_starts(data, c, l1):
-    for l in range(l1, len(data)):
-        char = data[l]
+    for i in range(l1, len(data)):
```

Comment:
I would say `l1` is also a bad variable name, it does look a lot like the number 11.